lib/StripeMethod.js

```diff
@@ -20,19 +22,26 @@ const makeAutoPaginationMethods = require('./autoPagination')
  * @param [spec.host] Hostname for the request.
  */
 function stripeMethod(spec) {
+  if (!!spec.path && !!spec.fullPath) {
+    throw new Error(
+      `Method spec specified both a 'path' (${spec.path}) and a 'fullPath' (${spec.fullPath}).`
+    );
+  }
   return function(...args) {
     const callback = typeof args[args.length - 1] == 'function' && args.pop();
 
-    spec.urlParams = utils.extractUrlParams(
-      this.createResourcePathWithSymbols(spec.path || '')
-    );
+    spec.urlParams = !!spec.fullPath
+      ? []
```

Comment:
`fullPath` is guaranteed not to have url params?

Reply:
Fixed!